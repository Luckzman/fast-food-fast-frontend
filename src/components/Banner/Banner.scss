/* SHOWCASE SECTION */
.showcase {
  height: 70vh;
  background-image: url(../../../assets/images/image-12.jpg);
  background-position: center;
  background-size: cover;
  background-repeat: no-repeat;
}
.showcase .dark-overlay {
  background-color: rgba(0, 0, 0, 0.6);
  height: 70vh;
  width: 100%;
}
.showcase .showcase-container {
  height: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
}
.showcase .showcase-ad {
  width: 80%;
  text-align: center;
  margin: auto;
}
.showcase h1 {
  font-family: fantasy;
  text-transform: uppercase;
  font-size: 3.5rem;
  color: #eee;
  line-height: 4.5rem;
}
.showcase p {
  color: #eee;
  font-family: "M PLUS Rounded 1c", sans-serif;
  padding: 10px 0;
}
.showcase-input {
  margin-top: 20px;
  line-height: 3rem;
}
.showcase-input input {
  border: none;
  background-color: transparent;
  color: #eee;
  font-size: 1.2rem;
  border-bottom: 1px solid #ccc;
}
.showcase-input input:focus {
  outline-color: transparent;
  border-bottom: 1px solid #00be29;
}
.showcase-input button {
  background-color: #00be29;
  color: #eee;
  padding: 5px;
  font-size: 1.2rem;
  border: none;
  border-radius: 10px;
  margin-left: 10px;
  cursor: pointer;
}
