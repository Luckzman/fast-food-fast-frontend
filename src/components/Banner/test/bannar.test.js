import React from 'react';
import { shallow } from 'enzyme';
import Banner from '..';

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Banner />);
    expect(wrapper).toMatchSnapshot();
  });
});
