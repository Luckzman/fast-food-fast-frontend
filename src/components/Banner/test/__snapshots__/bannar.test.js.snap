// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div
  className="showcase"
>
  <div
    className="dark-overlay"
  >
    <div
      className="showcase-container"
    >
      <div
        className="showcase-ad"
      >
        <h1>
          you order! we deliver!!
        </h1>
        <p>
          We're No.1 food delivery service in Nigeria!
        </p>
        <div
          className="showcase-input"
        >
          <input
            placeholder="Search Food"
            type="text"
          />
          <Link
            replace={false}
            to="/catalog"
          >
            <button
              type="button"
            >
              Search
            </button>
          </Link>
        </div>
      </div>
    </div>
  </div>
</div>
`;
